jumpscale/sals/reservation_chatflow/reservation_chatflow.py

```diff
@@ -1427,14 +1427,20 @@ def filter_disallowed_nodes(disallowed_node_ids, nodes):
             )
             nodes = filter_disallowed_nodes(disallowed_node_ids, nodes)
             nodes = self.filter_nodes(nodes, currency == "FreeTFT", ip_version=ip_version)
+            err_msg = f"""Failed to find resources for this reservation.
+                        If you are using a low resources environment like testnet, 
+                        please make sure to allow over provisioning from the settings tab in dashboard. 
+                        For more info visit <a href='https://manual2.threefold.io/#/3bot_settings?id=developers-options'>our manual</a><br>
+                        query params: cru={cru}, sru={sru}, mru={mru}, hru={hru}, pool_id={pool_id}
```

Comment:
maybe for users who don't know what query params is
```suggestion
                        requested resources: cru={cru}, sru={sru}, mru={mru}, hru={hru}, pool_id={pool_id}
```